guide/src/documentation/configuration/configure-hermes.md

```diff
@@ -22,6 +22,20 @@ hermes [--config CONFIG_FILE] COMMAND
 
 ## Configuration
 
+### Automatically Generating A Config File 
+
+The simplest way to configure Hermes for a given chain is by running the command
+
+```shell 
+hermes config auto
```

Comment:
It might be better to use something along the lines of `{{#template ../../templates/commands/hermes/config/auto_1.md PATH=~/<OUTPUT_PATH>/config.toml CHAIN_NAME:OPTIONAL_KEY_NAME=<CHAIN_1> <CHAIN_2>}}`

Reply:
[fe33d0f](https://github.com/informalsystems/hermes/pull/3301/commits/fe33d0fcf058522eda3042eb81a91463de69378d)